packages/just-scripts-utils/src/logger.ts

```diff
@@ -1,26 +1,48 @@
 import chalk from 'chalk';
+import yargs from 'yargs';
 
-function getTimestamp() {
+function logInternal(method: 'info' | 'warn' | 'error', symbol: string, ...args: any[]) {
   const now = new Date();
-  return `[${now.toLocaleTimeString()}]`;
+  const timestamp = chalk.gray(`[${now.toLocaleTimeString()}]`);
+
+  console[method](timestamp, symbol, ...args);
 }
 
 export interface ILogger {
-  info(msg?: any, ...optionalParams: any[]): void;
-  warn(msg?: any, ...optionalParams: any[]): void;
-  error(msg?: any, ...optionalParams: any[]): void;
+  /** Whether verbose logging is enabled. Default false unless --verbose arg is given. */
+  enableVerbose: boolean;
+  /** Log to `console.info` with a timestamp, but only if verbose logging is enabled. */
+  verbose(...args: any[]): void;
```

Comment:
Do you have a use case in mind? I hadn't done anything here that required this extra level of debugging, but WOULD be useful nonetheless.

Reply:
I remember wanting it for something when working on the fabric build scripts (don't remember the exact reason). Guessing if I wanted it before I'll want it again at some point.